packages/core/src/search/search.ts

```diff
@@ -337,6 +364,34 @@ function parseModifier(modifier: string): Operator {
   return MODIFIER_OPERATORS[modifier] || Operator.EQUALS;
 }
 
+function parseIncludeTarget(input: string): IncludeTarget {
+  const parts = input.split(':');
+
+  parts.forEach((p) => {
+    if (p === '*') {
+      throw new Error(`'*' is not supported as a value for search inclusion parameters`);
+    }
+  });
+
+  if (parts.length === 1) {
+    // Full wildcard, not currently supported
+    throw new Error(`Invalid include value '${input}': must be of the form ResourceType:search-parameter`);
```

Comment:
For errors, we try to use `OperationOutcomeError`

For example:

```ts
throw new OperationOutcomeError(badRequest('Invalid ...'));
```

This provides some extra structure:
* On the server, it can set the appropriate HTTP status codes from the `OperationOutcome`
* On the client side, a bunch of the UI elements use `OperationOutcome.issue.expression` to show the error next to the appropriate UI element

I'm not 100% sure what will happen here, but I suspect it has risk of HTTP 500?  Depends on context and if we're `try/catch`ing upstream.